### DISCUSSION TITLE
Jacobian and Hessian coordinates i,j

### DISCUSSION THREAD

**Discussion by camolina2** [lululxvi/deepxde#773] [Q&A]
Hi Lulu,

Thank you again for such amazing library!

I am confused with the meaning of derivates in hessian and jacobian coordinates (i, j), for example, if u is 1D, the values of i and j are coherent with the sintaxis?

`def pde(x, u):`
      `du_x = dde.grad.jacobian(u, x, i=0, j=0)`
      `du_y = dde.grad.jacobian(u, x, i=0, j=1)`
      `du_xx = dde.grad.hessian(u, x, i=0, j=0)`
      `du_yy = dde.grad.hessian(u, x, i=0, j=1)`
    `return ( v_1*du_x + v_2*du_y +  v_1_2* u +D*(du_xx + du_yy))`

By other hand, the constant v_1, v_2, v_1_2 and D should have and influence in the solution of the pde right?

Finally, the 'u' in the return term, is the same value as u? or inside is changing?

Thank you soooo much in advance!

**Comment by lululxvi** [comment]
See https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#module-deepxde.gradients

Yes for other questions.
